server/broadcast.go

```diff
@@ -386,6 +386,19 @@ func processSegment(cxn *rtmpConnection, seg *stream.HLSSegment) ([]string, erro
 	if ros != nil {
 		go func() {
 			now := time.Now()
+			hasZeroVideoFrame, err := ffmpeg.HasZeroVideoFrameBytes(seg.Data)
+			if err != nil {
+				glog.Warningf("Error checking for zero video frame nonce=%d manifestID=%s name=%s bytes=%d took=%s err=%v",
+					nonce, mid, name, len(seg.Data), time.Since(now), err)
+			} else {
+				glog.Infof("Checking for zero video frame nonce=%d manifestID=%s name=%s bytes=%d took=%s returned res=%v",
```

Comment:
This'll log every segment, yeah? Seems like it could be a `glog.Debugf` at least.

Reply:
Good point. Moved it around so it will show only for 0-frame segments.